recipe/meta.yaml

```diff
@@ -14,13 +14,15 @@ source:
 
 build:
   number: 0
+  skip:  # [py<36]
```

Comment:
Typo.  You missed the "True" ... expression is 'skip: True. # [py<36]'

Reply:
the typo on the skip section was corrected